### DISCUSSION TITLE
[SOLVED] Nuvie videoclips work with some C=64 and not with others

### DISCUSSION THREAD

**Discussion by giobbino** [frntc/RAD#52] [Q&A]
Hi all,
I've built a couple of RAD with Pi Zero 2W; one of them is just for Doom/Doom2 etc., and the other is for general purpose RAD/REU/etc.

I have some C=64 here; breadbin, C64C, etc.; all of them are PAL.

I've found that the nuvie clips only work with some C64 and not with the others. In example, they work with a breadbin and a C64C, but not with another breadbin and with my Sixty Clone (that is basically a breadbin).

On those machines the nuvie clips show glitches, artifacts and usually hang at certain point.

Nice fact: Doom and Sonic work fine with all of them.

Any idea? Something I could test?

**Comment by frntc** [comment]
Each C64 might require some changes in the bus timings (I'll revisit this at some point...). On the C64s where it's not working, try playing with those values. You can use the pound-sign in the RAD-menu, then adjust values until you see absolutely no glitches -- then you should be close or at working values.

Nuvies crash, because the REU functionality is corrupted with non-working timings. Doom doesn't care, because the 6510 is halted and can't crash ;-) (Sonic, afaik, only streams (mostly) graphics data, so no problem if that part has a faulty byte from time to time ;-))

**Reply by amiga23** [reply]
I do have a C64 with "PCB ASSY NO.250469 PCB NO.2523II REV.4". With default settings the letters in the main menu do flicker, NUVIEs do crash, C64OS also crashes during boot, when launching initial App. If I do set the "READ BA/DMA WRITE" to 472, then letters stop flicker, C64OS works fine, NUVIEs do play without GFX issues (as far as I can see), but the sound in NUVIEs in sometimes a bit distorted.
I do have another C64 with "PCB ASSY NO.250469 PCB NO.2523II REV.3" which works totally fine with default settings.

**Reply by giobbino** [reply]
I still have another C64 (probably assy 250407) that still doesn't work with the new settings (see below). I didn't care too much, since RAD work well enough on most of the C64s I use, but it seems you need to find a good setup for every C64. I also assumed what works fine on one C64 may not work on another. It's just a matter of "change setting and try until you find the right setup for _that_ C64.

**Comment by giobbino** [comment]
First of all, I wish to apologize because you wrote exactly about this issue in the readme, but I totally missed that (actually I hate when I spend time writing instructions and then the people don't read and ask, so I am ashamed when I found myself doing that LOL)

I changed ENABLE RW+ADDR from 467 to 437 and ENABLE DATA from 675 to 655, and now the nuvies work fine on my Sixty Clone (250407 replica) and also on my breadbin (possibly 250407 too), a C64C and a C64G (the beige, German edition - not sure what assy is). 
So maybe these values are more tolerant than the default.

**Comment by futureretrofusion** [comment]
Do koala videos play normally?

**Comment by giobbino** [comment]
Sorry, actually I know nothing about Koala videos, but if you want to pass me any link/instruction where I could download (and learn about) Koala videos, I will check it out.

**Comment by futureretrofusion** [comment]
Here is an example of a koala video... They are Sid chip specific but give
you video and sound...

https://youtu.be/7I4aiUJG3XI?si=6QUK-706d8VyCYj3

**Comment by giobbino** [comment]
Amazing, I didn't know. I got the koalaplayer v.04 from https://csdb.dk/release/?id=243459 and some koala video from the dropbox link in the readme inside the zip, and I can confirm they work perfectly. 

Awesome, I didn't imagine you can achive such audio performance from the SID.

Thanks for let me know about that, I will try to make some koala video using the tutorial/instruction from the zip file!
